src/js/constants/selectors-map.ts

```diff
@@ -110,6 +110,10 @@ export const visiblePassword = {
   visiblePassword: '.js-visible-password',
 };
 
+export const dropdownToggles = {
+  dropdownToggles: '.dropdown .dropdown-toggle[data-depth]',
```

Comment:
Ohoh, didn't see that, the only security there is the data-attribute, I would add a more specific menu selector at the beginning to make it a bit more secure, what do you think?

Reply:
.submenu?

Reply:
`.js-menu-desktop .dropdown .dropdown-toggle[data-depth]`
Like this?

Btw `data-depth` only used in main menu.

Reply:
Damn, didn't see we had a `js-` selector, I guess this is even more secure than what we talked in private @rodriciru

Reply:
I see no difference, but no problem. Committed